tutorial/tutorial-template-guide.md

```diff
@@ -0,0 +1,132 @@
+
+If you're writing a tutorial that's focused on learning how a particular feature works, you might describe that feature here. For example, a tutorial around using Microsoft Visio's templates to build flowcharts might start with an explanation of how flowcharts are used or what flowchart shapes represent. 
+
+If you're writing a tutorial where you provide users with a starter project, you could describe the project hierarchy here. For example, a tutorial around building a website with HTML, CSS, and JavaScript might explain how the folders are organized, what code is provided for the user, or what they need to add.
+
+### About the "Before you begin" section
+
+Use this section to list any prerequisites for completing the tutorial. This may include operating systems, languages, package managers, or software.
+
+### About the "Steps" section
+
+The steps section is where you describe what the user needs to do to complete the tutorial. The template organizes steps in the following way:
+
+```
+1. {Write the command here. Start with a verb.}
+
+   {Optional: Explanatory text}
+
+   {Optional: Code sample or screenshot that helps your users complete this step}
+
+   {Optional: Result}
+
+```
+
+The first element is the step itself, like "Enter a name for your new repository." When writing steps, keep these tips in mind:
+
+* Start with an imperative verb. For example, "connect", "set up", or "arrange." Don't use the *-ing* form of the verb because it is harder to translate.
+* Express steps as a complete thought. Instead of saying, "Set up access", you might say, "Set up access to a Cloud Storage bucket."
+
+Next, you may choose to add some additional information about the step. Continuing with the example, you might provide some best practices for creating memorable repository names.
+
+Optionally, you can add a code sample or screenshot after the explanatory text, depending on the type of tutorial you're writing. Screenshots are a great way to show specific parts of the screen you are referring to in a step. 
+
+If you're including code samples in your repository, make sure to add comments to your code to help learners understand what each part of the code does. Additionally, make sure your code samples work, as learners will copy and paste your code to use in their own projects.
+
+The last, and optional, element in a step, is the result. Continuing with the example, you might describe what happens after a user click the "Create repository" button.
+
+Here is an example step:
+
+```
+1. Enter a name for your new repository.   
+
+   Good repository names are short and self-explanatory. Avoid repository names with three or more words.
+   
+   After you click "Create repository", GitHub creates your repository and the main page for the repository is displayed.
+```
+
+If needed, you can add substeps below a primary step. Make sure to indent the substep one tab space over. 
```

Comment:
**Suggestion (future-proofing, non-blocking):** The suggestion to insert a tab space is particular to Markdown. So maybe say, "Make sure to indent the substep one tab space over if your tutorial is written in Markdown."